src/main/java/me/zeroeightsix/kami/gui/hudgui/elements/client/ModuleList.kt

```diff
@@ -132,7 +130,7 @@ object ModuleList : HudElement(
                 val color = ColorConverter.hexToRgb(Color.HSBtoRGB(hue, primaryHsb[1], primaryHsb[2]))
 
                 TextComponent.TextLine(" ").run {
-                    add(TextComponent.TextElement(module.name, color))
+                    add(TextComponent.TextElement(module.name.value, color))
```

Comment:
You should use delegates to get the value, much like we do for settings already.

https://github.com/kami-blue/client/blob/cbaee402d7b482be7204d657ac6b552c8eed0434/src/main/java/me/zeroeightsix/kami/setting/settings/AbstractSetting.kt#L23